src/App.tsx

```diff
@@ -10,33 +11,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const selectedTabId = (idValue: string) => {
+    const foundedTab = tabs.find((tab) => tab.id === idValue);
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+    if (foundedTab) {
+      setSelectedTab(foundedTab);
+    }
+  };
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+  return (
+    <div className="section">
+      <h1 className="title">{`Selected tab is ${selectedTab.title}`}</h1>
 
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <TabList
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/71132144/8f287ed4-4e32-4bc8-bab0-00f324b4639e)

Reply:
naming is still incorrect in some places, name it according to task description